services-custom/s3-transfer-manager/src/test/java/software/amazon/awssdk/transfer/s3/internal/S3TransferManagerTest.java

```diff
@@ -332,8 +339,15 @@ void downloadDirectory_throwException_shouldCompleteFutureExceptionally() {
     void close_shouldCloseUnderlyingResources() {
         S3TransferManager transferManager = new DefaultS3TransferManager(mockS3Crt, uploadDirectoryHelper, configuration, downloadDirectoryHelper);
         transferManager.close();
-        verify(mockS3Crt).close();
-        verify(configuration).close();
```

Comment:
Is there any reason this line is removed?

Reply:
removed by mistake.. added it back